scripts/lint-changelogs.js

```diff
@@ -0,0 +1,66 @@
+/* global __dirname process */
```

Comment:
This little script shares some stuff with the tests, we may eventually want to make a little changelog validation util package but for now I think it's OK